libvast/test/format/json.cpp

```diff
@@ -118,23 +118,96 @@ TEST(json to data) {
   CHECK_EQUAL(materialize(slice.at(0, 17)), data{reference});
 }
 
-#if 0
-TEST(json suricata) {
-  using reader_type = format::json::reader<format::json::suricata_selector>;
-  auto input = std::make_unique<std::istringstream>(std::string{eve_log});
-  reader_type reader{caf::settings{}, std::move(input)};
+TEST(json reader - suricata) {
+  using factory = vast::factory<format::reader>;
+  factory::initialize();
+  auto reader = factory::get("suricata")(caf::settings{});
+  auto input = std::make_unique<std::istringstream>(vast::test::EVE_JSON);
+  (*reader)->reset(std::move(input));
   std::vector<table_slice> slices;
   auto add_slice = [&](table_slice slice) {
     slices.emplace_back(std::move(slice));
   };
-  auto [err, num] = reader.read(2, 5, add_slice);
+  auto suricata_schema
+    = unbox(to<schema>(vast::test::BASE_SCHEMA + vast::test::SURICATA_SCHEMA));
+  (*reader)->schema(suricata_schema);
+  auto [err, num] = (*reader)->read(10, 10, add_slice);
   CHECK_EQUAL(err, ec::end_of_input);
-  REQUIRE_EQUAL(num, 2u);
-  CHECK_EQUAL(slices[0].columns(), 36u);
-  CHECK_EQUAL(slices[0].rows(), 2u);
-  CHECK(slices[0].at(0, 19) == data{count{4520}});
+  REQUIRE_EQUAL(num, 7u);
+  vast::table_slice::size_type NETFLOW_SLICE{0};
```

Comment:
Why is there all this `vast::`? As far as I can tell you're already in the namespace `vast` here.